acceptance/tests/parser_functions/hiera_in_templates.rb

```diff
@@ -2,7 +2,7 @@
 
 @module_name = "hieratest"
 @coderoot = master.tmpdir("#{@module_name}")
-@resultdir = agent.tmpdir("#{@module_name}_results")
+@resultdir = agents.first.tmpdir("#{@module_name}_results")
```

Comment:
Agree.  Feel free to close, just putting this up as an illustration of the issue.